roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -40,8 +40,9 @@ openshift_logging_es_pvc_prefix: "{{ openshift_hosted_logging_elasticsearch_pvc_
 # config the es plugin to write kibana index based on the index mode
 openshift_logging_elasticsearch_kibana_index_mode: 'unique'
 
-openshift_logging_elasticsearch_proxy_image_prefix: "openshift/oauth-proxy"
-openshift_logging_elasticsearch_proxy_image_version: "v1.0.0"
+openshift_logging_elasticsearch_proxy_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
+openshift_logging_elasticsearch_proxy_image_version: "{{ openshift_logging_image_version | default('latest') }}"
+#openshift_logging_elasticsearch_proxy_image_version: "v1.0.0"
```

Comment:
Which is the correct one to use here? Shouldn't we want to match all other components in the logging stack?

Reply:
I expect this to be tagged with all the other openshift images such that it is in lockstep with everything.  @jupierce ??

Reply:
@jcantrill We build oauth-proxy as part of each OCP build, so yes - lock step versioning as well as more generic tags like "v3.7".